ecdsa/resharing/round_3_old_step_2.go

```diff
@@ -24,7 +24,6 @@ func (round *round3) Start() *tss.Error {
 	if !round.ReSharingParams().IsOldCommittee() {
 		return nil
 	}
-	round.allOldOK()
```

Comment:
Doing this here would prevent _old-new_ players from waiting for messages from other players in the old committee. Namely, they would miss decommitments Dj generated in [round 1](https://github.com/timoth-y/tss-lib/blob/ecdsa/reshare_issues/ecdsa/resharing/round_1_old_step_1.go#L66-L69) by each other player Pj. 

This would lead to panic during decommiting in [round 4](https://github.com/timoth-y/tss-lib/blob/ecdsa/reshare_issues/ecdsa/resharing/round_4_new_step_2.go#L115).

Reply:
For reference: this works due to how the library implements the "wait" predicate [here](https://github.com/timoth-y/tss-lib/blob/ecdsa/reshare_issues/ecdsa/resharing/rounds.go#L70-L80).